pages/linux/toolbox-help.md

```diff
@@ -0,0 +1,12 @@
+# toolbox help
+
+> Displays help information about toolbox.
+> More information: <https://www.mankier.com/1/toolbox-help>.
+
+- Display the `toolbox` manual:
+
+`toolbox --help`
```

Comment:
Is `help` a subcommand or a flag? If it's a subcommand then why the `--`?

Reply:
help is a subcommand I will remove the flag from it.